### ISSUE TITLE
CaptureDragArea only works when already dragging

### ISSUE THREAD

**Issue by markdjohnston** [airbnb/visx#1575]
CaptureDragArea states "Whether to render an invisible rect below children to capture the drag area as defined by width and height." Currently, that's only the case if `drag.isDragging` is set to true, but that doesn't seem overly useful in most situations.

The issue stems here
https://github.com/airbnb/visx/blob/a3b79fd3bae63b100b1a8781f844631a2f3aa2ea/packages/visx-drag/src/Drag.tsx#L55

From my understanding this line should just be `{captureDragArea && (`

**Comment by MJ9001** [comment]
Note, I'm happy to make the PR myself to fix this.

**Comment by loichuder** [comment]
I agree. I have a usecase where I want to be able to drag a SVG line. As the line is a thin, adding an invisible rectangle on the top would make possible to start the drag from a bigger area for a better UX.

I also thought that `captureDragArea` would do this for me instead but the fact that the capturing rectangle only appears when dragging means it cannot be used for this.

Perhaps `captureDragArea` was meant for other purposes ?
